CHAT_BYLAWS.md

```diff
@@ -0,0 +1,49 @@
+# Apache Polaris - Project Chat Bylaws
+
+Apache Polaris is currently undergoing Incubation at the Apache Software Foundation.
+
+## Motivation
+
+The Polaris project uses a public and open chat service currently provided by Zulip hosted
+at https://polaris-catalog.zulipchat.com/. A few rules shall ensure that the chat conforms to the rules and best
+practices of the Apache Software Foundation and serves well as a collaboration tool for the project.
+
+Organizations and other open-source projects that contribute continuously and significantly to Polaris are welcome, but
+shall not use the Polaris chat as a vehicle for their own marketing without explicit approval of the project (P)PMC.
+
+## Code of Conduct
+
+The [Apache Software Foundation’s Code of Conduct](https://www.apache.org/foundation/policies/conduct.html) applies to
+the Polaris project public chat.
+
+## Governance
+
+* The Polaris project’s chat tool is not provided by the ASF - the Polaris project (P)PMC members govern and monitor the
+  chat service.
+* Everybody is welcome to join the Polaris project chat. Invites are not needed.
+* All users are automatically promoted to “members” (don’t stay in “guests”).
+* Polaris project (P)PMC members have “owner” privileges on the Zulip chat. Polaris project committers are granted
+  “moderator” or “administrator” privileges for moderation purposes.
+* This bylaws document shall be published on the project’s web site and linked from relevant public channels.
+* Only (P)PMC members are allowed to create new channels. The number of channels shall be limited to #general (user
+  discussions), #dev (development discussions), #announcements (release announcements), and #noise (GitHub feed).
+* Only (P)PMC members are allowed to notify a large number of users (aka @here).
```

Comment:
I would add an advise here to "invite" people to summarize important discussion on the dev mailing list.

Reply:
We can always direct people to go to the dev-ML. "Important" is something blurry - neither 100% objective nor subjective.